test/libsolidity/SolidityTypes.cpp

```diff
@@ -61,6 +61,10 @@ BOOST_AUTO_TEST_CASE(fixed_types)
 {
 	BOOST_CHECK(*Type::fromElementaryTypeName(ElementaryTypeNameToken(Token::Fixed, 0, 0)) == *make_shared<FixedPointType>(128, 18, FixedPointType::Modifier::Signed));
 	BOOST_CHECK(*Type::fromElementaryTypeName(ElementaryTypeNameToken(Token::UFixed, 0, 0)) == *make_shared<FixedPointType>(128, 18, FixedPointType::Modifier::Unsigned));
+
+	/// @todo add more comprehensive tests
+	BOOST_CHECK_EQUAL(FixedPointType::isValid(256, 77, 0), true);
```

Comment:
As the comment above says, there should indeed be some more tests about more corner cases at some point. However, for now: why use ``0`` and not ``false`` as third argument? And maybe it makes sense to already add tests for the signed 256-bit case as a reminder to also test the signed case more extensively later on.

Reply:
Changed.